src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -7,27 +7,29 @@
 import mate.academy.hibernate.relations.service.ActorService;
 import mate.academy.hibernate.relations.service.CountryService;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.service.impl.ActorServiceImpl;
+import mate.academy.hibernate.relations.service.impl.CountryServiceImpl;
+import mate.academy.hibernate.relations.service.impl.MovieServiceImpl;
 import mate.academy.hibernate.relations.util.HibernateUtil;
 import org.hibernate.SessionFactory;
 
 public class Main {
     public static void main(String[] args) {
-        // use this session factory when you will initialize service instances
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
-        Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
-        countryService.add(usa);
+        Country country = new Country("new zealand");
```

Comment:
Consider capitalizing the country name 'new zealand' to 'New Zealand' to maintain proper naming conventions.